src/Graphics/OpenGLContext/opengl_BufferedDrawer.cpp

```diff
@@ -1,13 +1,16 @@
 #include <Config.h>
 #include <CRC.h>
+#include <memory>
+#include <algorithm>
 #include "GLFunctions.h"
 #include "opengl_Attributes.h"
 #include "opengl_BufferedDrawer.h"
+#include "opengl_Wrapper.h"
 
 using namespace graphics;
 using namespace opengl;
 
-const u32 BufferedDrawer::m_bufMaxSize = 4194304;
+const u32 BufferedDrawer::m_bufMaxSize = 8*1024*1024;
```

Comment:
When I didn't have a limiter on how far ahead the core could get from the graphics API, eventually I would get graphics corruption due to this being too small.

Now that I have a limiter in place, I should be able to put this back to the original size.